### PR TITLE
Add reference to MiniApp Manifest

### PR DESCRIPTION
**Pull request by QingAn**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-lifecycle/pull/21.html" title="Last updated on Mar 21, 2022, 1:41 AM UTC (0369f1b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-lifecycle/21/8302e00...0369f1b.html" title="Last updated on Mar 21, 2022, 1:41 AM UTC (0369f1b)">Diff</a>

### CODE REVIEW

**Comment by QingAn** [comment]
This PR is to solve the comment from [w3ctag/design-reviews#523 (comment)](https://github.com/w3ctag/design-reviews/issues/523#issuecomment-928975772)
> 2. The lifecycle document doesn't reference Manifest at all. It was our understanding that Miniapps would use the manifest file with the Miniapp extensions and that doesn't seem to be referenced in the document.

**Comment by QingAn** [comment]
@xfq @espinr I have made the update, adding a note for clarification. Please have a check.

File: index.html [hunk 1]
```diff
@@ -318,6 +318,12 @@ <h3>
                 <p>
                     The <dfn>pagePath</dfn> attribute contains the page path for current MiniApp.
                 </p>
+                
+                <aside class="note">
+                    <p>
+                        By default, the pagePath uses the value defined by <a href="https://w3c.github.io/miniapp-manifest/#pages-member">the first item in the MiniApp manifest's pages member</a>. If the value is not set by <a href="https://w3c.github.io/miniapp-manifest/#pages-member"> the MiniApp manifest's pages member</a>, the pagePath uses the path of current MiniApp's first page.
```
Comment (espinr) [inline index.html]: Thanks @QingAn. Please consider changing the URL -> https://www.w3.org/TR/miniapp-manifest/#pages-member

**Review by espinr — COMMENTED** [review]
Not very relevant, and I can do it later. If you want, we can include the reference to the definition of the `pages` member. I've exported the `pages` member definition in w3c/miniapp-manifest@00b25a7ce0df922e56de94e076b617678ec3eff3 . Then we simply need to add this markup: 

```
<aside class="note" data-cite="MINIAPP-MANIFEST">
    ... the Manifest's [=pages=] member...

```

File: index.html (hunk 1 above)
Comment (QingAn) [reply index.html]: Thanks @espinr. The URL has been updated.

**Review by espinr — APPROVED** [review]
Thanks, @QingAn !

**Review by xfq — APPROVED** [review]

**Comment by QingAn** [comment]
Consensus to merge in MiniApp WG meeting 2022-03-24

**Pull request merged**